### ISSUE TITLE
Fabric won't Load

### ISSUE THREAD

**Issue by slenderknight1** [gorilla-devs/GDLauncher#943]
**Describe the bug**
the latest Fabric failed to launch it keeps on saying that it: Could not find required mod: fabric requires {java @ [>=16]}
![Capture](https://user-images.githubusercontent.com/22986932/121996544-a4795600-cd5d-11eb-8d89-798ec2266bce.PNG)

**Comment by RemagOfficial** [comment]
you need java 16

**Comment by casualmatt** [comment]
it's not possible to include java16 in the launcher it self as is done for the previous version of java ?

**Comment by slenderknight1** [comment]
I have the latest java installed so I don't know where the problem lies

**Comment by justTOBBI** [comment]
you need to point the java.exe for the latest version to it

**Comment by casualmatt** [comment]
yes but it's a minor chance for the project, and is on the idea of the project been simple. 

Now the java version that is downloaded from the launcher is specify in this json file: https://cdn.assets-gdevs.com/openjdk8.json
The support for java 16 is not just to be able to play on 1.17 but also for better performances on older version. 
The launcher could just allow the user to select the java version during setup or for pro pack. 

I love GDLauncher because i don't need to be the Support Guy in my group and connect remotely to help my friend to install a modpack or to just connect and play MC together.

**Comment by TheMarksmanXII** [comment]
Will GDLauncher ever change its "Automatic Setup" Java version from 8 to 16? And if so, then is there an ETA?

**Comment by TheMarksmanXII** [comment]
I finally found out what file can be used for the custom Java Path. If you have Windows and the vanilla Minecraft launcher installed, and you've used 1.17 at least once with it, then you can use it with GDLauncher like this:

1. Manage the Fabric 1.17.x instance
2. Enabled custom Java Path (if not already enabled)
3. Enter the following path as the Java Path:
C:\Program Files (x86)\Minecraft Launcher\runtime\java-runtime-alpha\windows-x64\java-runtime-alpha\bin\java.exe

Your instance now should launch normally, any crashes that may occur are likely because of mods or other items.
